src/metakb/cli.py

```diff
@@ -7,401 +7,366 @@
-        """Update Normalizer database.
-
-        :param str name: Name of the normalizer
-        :param bool load_normalizer: Whether or not to load normalizer db
-        :param CLI source_cli: Normalizer CLI class for loading and
-            deleting source data
-        """
-        if load_normalizer:
-            try:
-                echo_info(f"\nLoading {name} Normalizer data...")
-                source_cli.update_normalizer_db(["--update_all", "--update_merged"])
-                echo_info(f"Successfully Loaded {name} Normalizer data.\n")
-            except SystemExit as e:
-                if e.code != 0:
-                    raise e
-        else:
-            echo_info(f"{name} Normalizer is already loaded.\n")
-
-    @staticmethod
-    def _check_db_param(param: str, name: str) -> str:
-        """Check for MetaKB database parameter.
-        :param str param: value of parameter as received from command line
-        :param str name: name of parameter
-        :return: parameter value, or exit with error message if unavailable
-        """
-        if not param:
-            env_var_name = f"METAKB_DB_{name.upper()}"
-            if env_var_name in environ:
-                return environ[env_var_name]
-            # Default is local
-            if name == "URL":
-                return "bolt://localhost:7687"
-            if name == "username":
-                return "neo4j"
-            return "admin"
-        return param
-
-    @staticmethod
-    def _help_msg(msg: str = "") -> None:
-        """Handle invalid user input.
-        :param str msg: Error message to display to user.
-        """
-        ctx = click.get_current_context()
-        logger.fatal(msg)
-        if msg:
-            click.echo(msg)
-        else:
-            click.echo(ctx.get_help())
-        ctx.exit()
+    total_end = timer()
+    echo_info(
+        f"Successfully transformed all sources to CDM in "
+        f"{(total_end - total_start):.5f} s\n"
+    )
+
+
+def _load_normalizers_db() -> None:
+    """Load normalizer DynamoDB database source data."""
+    for name, update_normalizer_db_fn in [
+        ("Disease", update_normalizer_disease_db),
+        ("Therapy", update_normalizer_therapy_db),
+        ("Gene", update_normalizer_gene_db),
+    ]:
+        _update_normalizer_db(name, update_normalizer_db_fn)
+
+    echo_info("Normalizers database loaded.\n")
+
+
+def _update_normalizer_db(
+    name: str,
+    update_normalizer_db_fn: callable,
+) -> None:
+    """Update Normalizer DynamoDB database.
+
+    :param name: Name of the normalizer
+    :param update_normalizer_db_fn: Function to update the normalizer DynamoDB database
+    """
+    try:
+        echo_info(f"\nLoading {name} Normalizer data...")
+        update_normalizer_db_fn(["--update_all", "--update_merged"])
+        echo_info(f"Successfully Loaded {name} Normalizer data.\n")
+    except SystemExit as e:
+        if e.code != 0:
+            raise e
+
+
+@staticmethod
```

Comment:
```suggestion
```

Reply:
(I think I got all of them but it's a little awkward to ctrl+f in the PR review interface)